src/timearray.jl

```diff
@@ -64,14 +70,14 @@ isempty(ta::TimeArray) = (length(ta) == 0)
 
 ###### show #####################
 
-function show{T,N}(io::IO, ta::TimeArray{T,N})
+function show(io::IO, ta::TimeArray{T}) where {T}
```

Comment:
In particular, the type `TimeArray{T}` should have an implicit `where N`, since the `TimeArray` type has two type parameters, `T` and `N`, and both still exist. That is, it should be the case that `TimeArray{T} where T == TimeArray{T,N} where N where T`.